catalogue/webapp/pages/api/works/items.ts

```diff
@@ -1,41 +1,68 @@
 import fetch from 'isomorphic-unfetch';
-import { NextApiRequest, NextApiResponse } from 'next';
+import { NextApiResponse } from 'next';
 import { ItemsWork, CatalogueApiError } from '@weco/common/model/catalogue';
-import { catalogueApiError } from '../../../services/catalogue/common';
+import {
+  catalogueApiError,
+  rootUris,
+  globalApiOptions,
+} from '../../../services/catalogue/common';
 import hasOwnProperty from '@weco/common/utils/has-own-property';
+import { Toggles } from '@weco/toggles';
+import withToggles, {
+  NextApiRequestWithToggles,
+} from '@weco/common/api-routes-middleware/withToggles';
 
 export function isCatalogueApiError(
   response: ItemsWork | CatalogueApiError
 ): response is CatalogueApiError {
   return Boolean(hasOwnProperty(response, 'type') && response.type === 'Error');
 }
 
-async function fetchWorkItems(
-  id: string
-): Promise<ItemsWork | CatalogueApiError> {
+function getApiUrl(apiOptions, workId: string): string {
+  return `${rootUris[apiOptions.env]}/v2/works/${workId}/items`;
+}
+
+function getApiKey(apiOptions): string | undefined {
```

Comment:
```suggestion
function getApiKey(apiOptions): string {
```

It'll never be undefined

Reply:
Maybe it's having trouble peeking inside the `if` maybe drop the `else` and see if it make ts happy?

Reply:
No joy with that, but [can I tell Typescript what environment variables are available](https://github.com/wellcomecollection/wellcomecollection.org/blob/edcd9ac338133e32382eab02fefc2af81f11054d/catalogue/webapp/types/environment.d.ts#L3) and then it's happy